gpkit/tests/t_constraints.py

```diff
@@ -358,9 +358,23 @@ def test_substitution_issue905(self):
         sol = bm.solve(verbosity=0)
         self.assertAlmostEqual(sol["cost"], 1.0)
 
+class TestLocalsolve(unittest.TestCase):
+    """Test different SP solution methods"""
+
+    def test_penaltyccp(self):
+        x = Variable("x")
+        y = Variable("y", 2.)
+        z = Variable("z")
+        with SignomialsEnabled():
+            m = Model(1/z, [z <= x**2 + y, x*z == 2])
+        sol = m.localsolve(verbosity=0)
+        m.program.relax = True
+        sol_pccp = m.localsolve(verbosity=0, relax=True)
+        self.assertEqual(len(m.program.gps[-1].varkeys), 3)
+        self.assertAlmostEqual(sol['cost'], sol_pccp['cost'])
```

Comment:
🙌  hooray for unit tests

Reply:
This makes it seem like we do not have tests for vanilla `localsolve()`, but those are implemented in `t_model.py`. We should decide where these tests belong and put them all in the same place? The tests in `t_model` also give access to `MULTI_SOLVER_TESTS` to run for all solvers, which is probably appropriate for testing `penalty_ccp`.